locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed using
```

Comment:
Please strongly stress using the sole supported installation method - the curl command in the readme - over alternative unsupported methods of installation :-)